open-craft/edx-learndot
# Updates LearnDot enrolment records when a learner passes an Open edX course

Initial integration between LearnDot and Open edX.

**Testing instructions**

1. Set up a master devstack, and follow [README#installation](https://github.com/open-craft/edx-learndot/blob/091a313021ce09b6503d44941b3c79ee49740599/README.rst#installation) instructions.
1. Add a valid `LEARNDOT_API_BASE_URL` to the `lms.env.json`.
1. Add a valid `LEARNDOT_API_KEY` to `lms.auth.json`.
1. Create a course you can pass. A new course with one gradable unit that's 100% of the grade would work.
1. Configure a CourseMapping in the Django admin, using a valid Learndot component ID and the course key from the course to be passed.
1. Enroll a learner in the course, complete the gradable unit as that learner, and check that the Learndot enrolment status was set to "PASSED". This will be in the devstack LMS logs, and can also be verified via the Learndot API, using cURL.

**Author**: @ciuin 

**Reviewer**
